test/tests/functional/pbs_compress.py

```diff
@@ -60,8 +60,6 @@ def test_submit_job_comp(self):
         j.set_attributes({ATTR_v: "ABC=%s" % data})
         jid = self.server.submit(j)
         self.server.expect(JOB, {'job_state': 'R'}, id=jid)
-        self.server.log_match("tpp_deflate called inside tpp_send routine")
```

Comment:
@dinesh1306 Can you please change test suite name also? its not as per our PTL guidelines of test suite name. Thx!